wat2wasm4cpp.py

```diff
@@ -0,0 +1,131 @@
+#!/usr/bin/python3
+
+"""wat2wasm4cpp
+
+This script converts C++ comments containing WebAssembly text format (WAT)
+into WebAssembly binary format.
+
+It uses wat2wasm tool from WABT (https://github.com/WebAssembly/wabt)
+and expect the tool be available in PATH.
+- On Linux/Debian wabt package is avialble.
+- On macos wabt can be installed via homebrew.
+
+It searches for following block comments:
+
+    /* Additional description or wat2wasm options.
+    (module
+      (func ...)
+    )
+    */
+
+Inportantly, the first line of the comment can contain additional wat2wasm
```

Comment:
```suggestion
Importantly, the first line of the comment can contain additional wat2wasm
```

Reply:
Not quite clear whether I can combine a regular comment like in the previous snippet with `wat2wasm` comment line?